src/Magnum/Audio/Buffer.h

```diff
@@ -102,6 +104,57 @@ class Buffer {
             return *this;
         }
 
+        /**
+         * @brief Set buffer loop points
+         * @param loopStart The loop's start point in samples
+         * @param loopEnd   The loop's end point in samples
+         * @return Reference to self (for method chaining)
+         *
+         * @requires_al_extension Extension @al_extension{SOFT,loop_points}
+         */
+        Buffer& setLoopPoints(Int loopStart, Int loopEnd);
+
+        /**
+         * @brief Set buffer to loop from the beginning until a certain point
+         *
+         * Equivalent to calling @ref setLoopPoints() with @p loopStart equal to
+         * 0.
+         * @param loopEnd   The loop's end point in samples
+         * @return Reference to self (for method chaining)
```

Comment:
`@param` and `@return` goes under `@brief`, detailed docs go after that.

Reply:
In some of the docs, the detailed docs were above. I think `Buffer.h` itself has an example.

Reply:
Changed locally.